Sources/Talkshoplive/Core/Chat/ChatProvider.swift

```diff
@@ -98,11 +106,11 @@ public class ChatProvider {
             case .success(let apiResponse):
                 // Set the details and invoke the completion with success.
                 if let eventId = apiResponse.id {
-                    let publicChannel = "chat.\(eventId)"
+                    self.channelToPublish = "chat.\(eventId)"
                     let eventsChannel = "events.\(eventId)"
```

Comment:
`eventsChannel` Can you save this to state as well? We will need it later in other functions.